docs/install.md

```diff
@@ -70,6 +70,7 @@ to get started.
 
     $ pip install docker-compose
 
+> **Note:** pip version 6.0 or greater is required
```

Comment:
Kind of odd to use a blockquote marker to emphasize this line. It's probably fine without it (and more semantically sound)?